external/wildfly/README.md

```diff
@@ -0,0 +1,31 @@
+# External Wildfly Tests
+
+Wildfly tests are part of the external third-party application tests that help verify that the Adoptium binaries are good by running a variety of Java applications inside of Docker containers. AdoptOpenJDK/openjdk-tests/Issue [#172](https://github.com/AdoptOpenJDK/openjdk-tests/issues/172) lists the applications that we have initially targeted to best exercise the Adoptium binaries. For each application, we choose to run a selection of their functional tests. Wildfly tests are pulled from the [wildfly-test-suite](https://github.com/wildfly/wildfly.git) repository.
```

Comment:
In last sentence, change wildfly-test-suite to wildfly (since we are using the wildfly repo)

Reply:
@smlambert I have changed the last line to wildfly in place of wildfly-test-suite.